batch/create-batch-tables.sql

```diff
@@ -173,13 +173,13 @@ BEGIN
 
   SET msec_mcpu_diff = msec_diff * job_cores_mcpu;
 
-  UPDATE batches
```

Comment:
Can you add a comment about the organizing principle somewhere?  It isn't immediately obvious to me.

Reply:
OK, I added some information. I admittedly don't have good data on how these changes affect the performance of batch.